regexes/client/hints/browsers.yml

```diff
@@ -142,3 +142,8 @@
 'com.skyfire.browser': 'Skyfire'
 'com.sonymobile.smallbrowser': 'Sony Small Browser'
 'org.hola': 'hola! Browser'
+'it.ideasolutions.amerigo': 'Amerigo'
+'org.xbrowser.prosuperfast': 'Internet Browser Private'
```

Comment:
I think it would be better if we use an alternative name
`xBrowser Pro Super Fast`